### DISCUSSION TITLE
Problem when I print with an atlas

### DISCUSSION THREAD

**Discussion by GlaDal** [opengisch/QField#3861] [Q&A]
Hi, I need to understand the origin of my problem to find the appropriate solution.
I configured my Qfield project in Qgis 3.22 where I define an atlas with a name based on an expression which work at the export (the name is well configured in the pdf export).
But, when I put this project on Qfield (with QfieldCloud), the name of my pdf is wrong. I obtain a default name like this : 'MapTest-20230131_143205' (wich is the name of the layout configured in Qgis and date / hour). The PDF is automaticcaly opened in Adobe reader
I tried on the same layout with a basic expression like 'output_'||'$id', It worked on Qgis but not in Qfield. 
I tried on a new layout with only an object map with the same basic expression. It worked on Qgis and Qfield. The PDF in stocked on Qfield's project folder and not opened automatically by Adobe Reader.
I tried on this new layout using with the 'complex' expression (it's a concatenation). It worked on Qgis and Qfield. 
My atlas is based on a geopackage with only one feature for testing.
I need to understand why my complex expression worked on Qgis but not in Qfield. Thanks !

**Comment by meyerlor** [comment]
A sample project would certainly helpful.
